jekyll/_cci2/migrating-from-launch-agent-to-the-new-machine-runner-agent.adoc

```diff
@@ -0,0 +1,165 @@
+```
+
++
+This will remove all launch agent components, including the configuration file.
+
++
+If you want to reuse the values with the new install, you should save the file to a temporary location while migrating. It is stored by default at `$env:ProgramFiles\CircleCI`
+
+
+[#install-machine-agent]
+== 2. Installing the Machine Agent
+
+Packaged versions of the machine runner agent are available for Linux and MacOS for a streamlined installation experience.
+
+[#install-machine-agent-linux]
+=== Linux
+
+. At present deb and rpm packages are available for linux. Use the following commands to install them
+
++
+```shell
+# DEB Commands
+# This script installs the package repository to the package manager
+curl -s https://packagecloud.io/install/repositories/circleci/runner/script.deb.sh | sudo bash
+sudo apt install circleci-runner
+```
+
++
+```shell
+# RPM Commands
+# This script installs the package repository to the package manager
+curl -s https://packagecloud.io/install/repositories/circleci/runner/script.rpm.sh | sudo bash
+sudo yum install circleci-runner
+```
+
++
+The packages will install the latest version of the agent. They also create a circleci user as well as required directories for operation of the agent.
+
+. The packages will create a template configuration file at `/etc/circleci-runner/circleci-runner-config.yaml`. This configuration is 1:1 compatible with the launch agent configuration, apart from the auto-update feature. Since auto-update has been removed from the new agent, you should use the appropriate package management update tools to get new versions of the package when they are released. When using the `apt` package to update the agent package, user interaction will be required to clear the configuration file modified prompt. This can be avoided using the `--force-confold` flag or by removing the modified configuration file before upgrading. 
+
++
+Once the config file has been populated, the following command will restart the agent so it can begin claiming:
+
++
+```shell
+sudo systemctl enable circlci-runner
+sudo systemctl start circleci-runner
+```
+
+[#install-machine-agent-macos]
+=== MacOS
+
+. A `brew` package is available for MacOS. Use the following commands to install the machine agent:
+
++
+```shell
+brew tap circleci-public/homebrew-circleci
+brew install --cask circleci-public/homebrew-circleci/circleci-runner
+```
+
++
+The package will install the latest version of the agent along with required directories for the operation of the agent.
+
+. The package will create a template configuration file (if it does not already exist) at `$HOME/Library/Preferences/com.circleci.runner/config.yaml`. This configuration is 1:1 compatible with the launch agent configuration, apart from the auto-update feature. Since auto-update has been removed from the new agent, you should use the appropriate package management update tools to get new versions of the package when they are released.
```

Comment:
```suggestion
. The package creates a template configuration file (if it does not already exist) at `$HOME/Library/Preferences/com.circleci.runner/config.yaml`. This configuration is 1:1 compatible with the launch agent configuration, apart from the auto-update feature. Since auto-update has been removed from the new agent, you should use the appropriate package management update tools to get new versions of the package when they are released.
```